pkg/apis/migration/v1alpha1/migplan_types.go

```diff
@@ -416,19 +343,19 @@ func (r *MigPlan) UpdateRegistryDC(storage *MigStorage, deploymentconfig *appsv1
 		}
 		envFrom = append(envFrom, source)
 	}
-	deploymentconfig.Spec = appsv1.DeploymentConfigSpec{
-		Replicas: 1,
-		Selector: map[string]string{
-			"app":              name,
-			"deploymentconfig": name,
-		},
-		Strategy: appsv1.DeploymentStrategy{Resources: kapi.ResourceRequirements{}},
-		Template: &kapi.PodTemplateSpec{
+
+	deployment.Spec = appsv1.DeploymentSpec{
+		Replicas: pointer.Int32Ptr(1),
+		Selector: metav1.SetAsLabelSelector(map[string]string{
+			"app":        name,
+			"deployment": name,
```

Comment:
@shubham-pampattiwar you can add "migplan: r.uid" here to get the label on the pods

Reply:
Done